server/translations.json

```diff
@@ -375,5 +375,43 @@
         "named_private_board_description": "您还可以创建一个命名的私有白板，它有一个自定义的URL，所有知道它名字的人都可以访问它。",
         "board_name_placeholder": "白板名称",
         "view_source": "GitHub上的源代码"
+    },
+ "vn": {
+        "hand": "Tay",
+        "loading": "Đang tải",
+        "tagline": "Một công cụ vẽ cộng tác trực tuyến miễn phí và mã nguồn mở. Cùng nhau phác thảo những ý tưởng mới trên WBO!",
+        "configuration": "Cấu hình",
+        "collaborative_whiteboard": "Bảng trắng cộng tác",
+        "size": "Size",
```

Comment:
Is this Vietnamese ?

Reply:
yes. It's vietnamese

Reply:
Gimp seems to be using "cỡ". Would that be a better translation ?

The word `size` does not seem to be in the vietnamese dictionary...

https://dictionary.cambridge.org/dictionary/english-vietnamese/size

Reply:
It is widely used in Vietnam.
Everyone understands it

Reply:
no one used the word "cở"